### DISCUSSION TITLE
Weird Bug?

### DISCUSSION THREAD

**Discussion by mikki8** [blakeblackshear/frigate#15439] [General Support] [answered]
Hi,

encountering it now and then. GPU is starting to process something(nothing suspicious on cam or else)
![grafik](https://github.com/user-attachments/assets/9922b2a4-b020-4977-b515-1105a4668228)
until the whole system is freezing.
![grafik](https://github.com/user-attachments/assets/ddcf291c-b3a9-4de9-b845-9b52bf32afd7)
![grafik](https://github.com/user-attachments/assets/ff5c7889-4083-4641-87ca-33e4e81a017f)
![grafik](https://github.com/user-attachments/assets/86a9c443-9aa8-4530-8b90-e094e3a077b0)
![grafik](https://github.com/user-attachments/assets/4921d816-4f60-4128-9363-19730a3ce301)

`frigate  | 2024-12-11 05:08:59.680082120  127.0.0.1 - - [11/Dec/2024:05:08:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:09:59.699980182  127.0.0.1 - - [11/Dec/2024:05:09:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:10:59.724341623  127.0.0.1 - - [11/Dec/2024:05:10:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:11:59.744397470  127.0.0.1 - - [11/Dec/2024:05:11:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:12:59.761946784  127.0.0.1 - - [11/Dec/2024:05:12:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:13:59.789077303  127.0.0.1 - - [11/Dec/2024:05:13:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:14:59.811996690  127.0.0.1 - - [11/Dec/2024:05:14:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:15:59.830272386  127.0.0.1 - - [11/Dec/2024:05:15:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:16:59.855862065  127.0.0.1 - - [11/Dec/2024:05:16:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:17:59.874451311  127.0.0.1 - - [11/Dec/2024:05:17:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:18:59.895978133  127.0.0.1 - - [11/Dec/2024:05:18:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:19:59.916285335  127.0.0.1 - - [11/Dec/2024:05:19:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:20:59.939715786  127.0.0.1 - - [11/Dec/2024:05:20:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:21:59.966145423  127.0.0.1 - - [11/Dec/2024:05:21:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:22:59.986760024  127.0.0.1 - - [11/Dec/2024:05:22:59 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:24:00.010645309  127.0.0.1 - - [11/Dec/2024:05:24:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:25:00.033520991  127.0.0.1 - - [11/Dec/2024:05:25:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:26:00.049604493  127.0.0.1 - - [11/Dec/2024:05:26:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:27:00.069796301  127.0.0.1 - - [11/Dec/2024:05:27:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:28:00.087072243  127.0.0.1 - - [11/Dec/2024:05:28:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:29:00.110019358  127.0.0.1 - - [11/Dec/2024:05:29:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:30:00.130672343  127.0.0.1 - - [11/Dec/2024:05:30:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:30:01.152898299  [h264 @ 0x7f49d01a9800] cabac decode of qscale diff failed at 91 68
frigate  | 2024-12-11 05:30:01.152902397  [h264 @ 0x7f49d01a9800] error while decoding MB 91 68, bytestream -5
frigate  | 2024-12-11 05:31:00.152146108  127.0.0.1 - - [11/Dec/2024:05:31:00 +0100] "" 400 0 "-" "-" "-"
frigate  | 2024-12-11 05:47:05.762470142  [mov,mp4,m4a,3gp,3g2,mj2 @ 0x7f49d00eb340] moov atom not found
frigate  | 2024-12-11 05:47:09.553597034  [ERROR:0@254679.015] global cap.cpp:164 open VIDEOIO(CV_IMAGES): raised OpenCV exception:
frigate  | 2024-12-11 05:47:09.570293671  
frigate  | 2024-12-11 05:47:09.570295765  OpenCV(4.9.0) /io/opencv/modules/videoio/src/cap_images.cpp:274: error: (-215:Assertion failed) number < max_number in function 'icvExtractPattern'
frigate  | 2024-12-11 05:47:09.570296477  
frigate  | 2024-12-11 05:47:09.570297048  
frigate  | 2024-12-11 05:48:05.744870463  [mov,mp4,m4a,3gp,3g2,mj2 @ 0x7f49d00eb340] moov atom not found
frigate  | 2024-12-11 05:48:06.133092364  [2024-12-11 05:48:04] watchdog.rtop-wohnzimmer       INFO    : Waiting for ffmpeg to exit gracefully...
frigate  | 2024-12-11 05:48:10.909046485  [ERROR:0@254740.004] global cap.cpp:164 open VIDEOIO(CV_IMAGES): raised OpenCV exception:
frigate  | 2024-12-11 05:48:10.924860879  
frigate  | 2024-12-11 05:48:10.924864346  OpenCV(4.9.0) /io/opencv/modules/videoio/src/cap_images.cpp:274: error: (-215:Assertion failed) number < max_number in function 'icvExtractPattern'
frigate  | 2024-12-11 05:48:10.924865348  
frigate  | 2024-12-11 05:48:10.924866239`

**Comment by hawkeye217** [comment]
These errors in your log indicate bad data coming from the camera.

**Reply by mikki8** [reply]
will ffmpeg7 maybe fix it ?

**Reply by NickM-27** [reply, marked as answer]
Some users have reported that some issues have been improved

**Reply by mikki8** [reply]
looks good so far - nearly 12 hours in without errors on ffmpeg7
![grafik](https://github.com/user-attachments/assets/e12103c7-8be8-4b95-8eef-6957efa935a1)
